modules/client/examples/02-multisig-client/11-get-members.ts

```diff
@@ -16,12 +16,12 @@ const contextPlugin: ContextPlugin = ContextPlugin.fromContext(context);
 // Instantiate a Multisig plugin client.
 const multisigClient: MultisigClient = new MultisigClient(contextPlugin);
 
-const daoAddressorEns: string = "0x1234548357023847502348"; // or my-dao.dao.eth
+const daoPluginAddress: string = "0x1234548357023847502348"; // The address of the plugin that DAO has installed. You can find this through calling `getDao(daoAddress)` and getting the DAO details .
```

Comment:
This is confusing because each DAO has its own specific plugin address. I wan to make sure that is clear in the naming beyond than in just the additional comments.